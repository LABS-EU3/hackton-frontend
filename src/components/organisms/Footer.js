import React from 'react';

import WideFooter from '../../components/atoms/WideFooter';
import FooterContainer from '../../components/atoms/FooterContainer';
import Group from '../../components/atoms/Group';
import SocialIcon from '../../components/atoms/SocialIcon';
import NavLink from '../../components/atoms/NavLink';
import Paragraph from '../../components/atoms/Paragraph';

import linkedin from '../../assets/Icon-linkedin.png';
import twitter from '../../assets/Icon-twitter.png';
import facebook from '../../assets/Icon-facebook.png';

const Footer = () => (
  <WideFooter>
    <FooterContainer>
      <Group>
        <SocialIcon src={linkedin} alt="LinkedIn" />
        <SocialIcon src={twitter} alt="Twitter" />
        <SocialIcon src={facebook} alt="Facebook" />
      </Group>

      <Group>
        <NavLink to="#">Team</NavLink>
        <NavLink to="#">Privacy</NavLink>
        <NavLink to="#">Contacts</NavLink>
      </Group>

      <Group>
        <Paragraph>
          International Crafters Â© 2020
        </Paragraph>
      </Group>
    </FooterContainer>
  </WideFooter>
);

export default Footer;
