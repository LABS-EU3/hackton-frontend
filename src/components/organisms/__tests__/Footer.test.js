import React from 'react'
import { render } from '@testing-library/react'
import Footer from '../Footer';


describe("It should render <Footer/> template correctly", () => {
  
    it("renders the Footer templatet correctly", () => {
      const template = render(<Footer/>)
      expect(template).toMatchSnapshot();
    });
  });