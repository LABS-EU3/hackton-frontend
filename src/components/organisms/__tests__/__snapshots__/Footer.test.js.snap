// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render <Footer/> template correctly renders the Footer templatet correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bdVaJa bDiKvh"
      >
        <div
          class="sc-bwzfXH ldOoJf"
        >
          <div>
            <img
              alt="LinkedIn"
              class="sc-htpNat hwcHdo"
              src="Icon-linkedin.png"
            />
            <img
              alt="Twitter"
              class="sc-htpNat hwcHdo"
              src="Icon-twitter.png"
            />
            <img
              alt="Facebook"
              class="sc-htpNat hwcHdo"
              src="Icon-facebook.png"
            />
          </div>
          <div>
            <a
              class="sc-EHOje sc-gzVnrw iKqYXF"
              href="#"
            >
              Team
            </a>
            <a
              class="sc-EHOje sc-gzVnrw iKqYXF"
              href="#"
            >
              Privacy
            </a>
            <a
              class="sc-EHOje sc-gzVnrw iKqYXF"
              href="#"
            >
              Contacts
            </a>
          </div>
          <div>
            <p
              class="sc-htoDjs sc-dnqmqq fMTgah"
            >
              International Crafters © 2020
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bdVaJa bDiKvh"
    >
      <div
        class="sc-bwzfXH ldOoJf"
      >
        <div>
          <img
            alt="LinkedIn"
            class="sc-htpNat hwcHdo"
            src="Icon-linkedin.png"
          />
          <img
            alt="Twitter"
            class="sc-htpNat hwcHdo"
            src="Icon-twitter.png"
          />
          <img
            alt="Facebook"
            class="sc-htpNat hwcHdo"
            src="Icon-facebook.png"
          />
        </div>
        <div>
          <a
            class="sc-EHOje sc-gzVnrw iKqYXF"
            href="#"
          >
            Team
          </a>
          <a
            class="sc-EHOje sc-gzVnrw iKqYXF"
            href="#"
          >
            Privacy
          </a>
          <a
            class="sc-EHOje sc-gzVnrw iKqYXF"
            href="#"
          >
            Contacts
          </a>
        </div>
        <div>
          <p
            class="sc-htoDjs sc-dnqmqq fMTgah"
          >
            International Crafters © 2020
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
